src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  Array.from(section.rows).forEach((row) => {
```

Comment:
Converting `HTMLCollection` to an array using `Array.from()` is not necessary here. `HTMLCollection` is an array-like object and you can use `forEach` directly on it. This will also improve the performance of your code.